website/src/components/expandable.svelte

```diff
@@ -1,5 +1,6 @@
 <script lang="ts">
 	import { faChevronDown } from "@fortawesome/pro-solid-svg-icons";
+	import { CaretDown } from "phosphor-svelte";
```

Comment:
I believe this is the incorrect way of importing.
https://github.com/haruaki07/phosphor-svelte?tab=readme-ov-file#usage

Reply:
I use this https://github.com/haruaki07/phosphor-svelte?tab=readme-ov-file#import-optimizer-experimental to rewrite the imports.